tests/e2e/specs/stories-editor/video-poster-image-extraction.test.js

```diff
@@ -76,18 +57,7 @@ describe.skip( 'Video Poster Image Extraction', () => { // eslint-disable-line j
 			// Click the media library button.
 			await page.waitForSelector( '.editor-media-placeholder__media-library-button' );
 			await page.click( '.editor-media-placeholder__media-library-button' );
-
-			// Wait for media modal to appear and upload video.
-			await page.waitForSelector( '.media-modal input[type=file]' );
-			const inputElement = await page.$( '.media-modal input[type=file]' );
-			const testImagePath = path.join( __dirname, '..', '..', 'assets', 'clothes-hanged-to-dry-1295231.mp4' );
-			const filename = uuid();
-			const tmpFileName = path.join( os.tmpdir(), filename + '.mp4' );
-			fs.copyFileSync( testImagePath, tmpFileName );
-			await inputElement.uploadFile( tmpFileName );
-
-			// Wait for upload.
-			await page.waitForSelector( `.media-modal li[aria-label="${ filename }"]` );
```

Comment:
This is extracted verbatim into `uploadMedia()`, so I could also use it in the new test file.

Reply:
It's actually not verbatim anymore, but very similar. It now returns the file name with the extension.